storage/ndb/rest-server2/server/src/fs_cache.cpp

```diff
@@ -352,7 +352,7 @@ void FSMetadataCache::cache_entry_updater(Uint32 key_cache_id) {
   while (true) {
     Uint32 sleepMillis = 100;
     Uint64 eviction_ms =
-      (Uint64)globalConfigs.security.apiKey.cacheUnusedEntriesEvictionMS;
+     (Uint64)globalConfigs.featureStore.featureStoreMetadataCache.cacheUnusedEntriesEvictionMS;
```

Comment:
You could take this opportunity to declare it `const` and move it out of the loop.